### PR TITLE
all edits - including new lessons

### PR DESCRIPTION
**Pull request by CaitlinBate**

### CODE REVIEW

File: _episodes/01-plone.md [hunk 1]
```diff
@@ -83,39 +85,39 @@ Editing a file:
...
 
 ### Detection Extracts
 
-OTN and its partner Nodes create Detection Extracts on a semi-regular basis (approximately every 4 months) following a cross-Node coordinated detection matching event known as a Data Push. These Detection Extract files contain only the detections for the year corresponding to the suffix on the file name. See the detailed [detection extract documentation](https://members.oceantrack.org/data/otn-detection-extract-documentation-matched-to-animals) for more information.
+OTN and its partner Nodes create Detection Extracts on a regular basis, every 4 months (Push months are February, June, and October) following a cross-Node coordinated detection matching event known as a Data Push. These Detection Extract files contain only the detections for the year corresponding to the suffix on the file name. See the detailed [detection extract documentation](https://members.oceantrack.org/data/otn-detection-extract-documentation-matched-to-animals) for more information.
```
Comment (slandovs) [inline _episodes/01-plone.md]: add a comma following the push months bracket

File: _episodes/01-plone.md [hunk 2]
```diff
@@ -83,39 +85,39 @@ Editing a file:
...
 
 ### Equipment
 
-This subfolder is the location where Shipping Lists can be posted and reviewed, if relevant. The shipping/inventory tracking workflow has moved to the [SnipeIT inventory system](https://ops.oceantrack.org/snipeit/).
+This subfolder is the location where Shipping Lists can be posted and reviewed, if relevant. The shipping/inventory tracking workflow has moved to the [SnipeIT inventory system](https://ops.oceantrack.org/snipeit/) and [Asana](https://app.asana.com/0/1203822208185209/1203822208185209).
 
 ### Background
 
-This subfolder is the location where OTN Project Management staff will upload loan agreements, project plans and legal paperwork. This folder is only visible to OTN staff members by default.
+This subfolder is the location where OTN Project Management staff will upload loan agreements, project plans and legal paperwork if relevant. This folder is only visible to OTN staff members by default.
```
Comment (slandovs) [inline _episodes/01-plone.md]: above you have used the oxford comma, so there should be a comma after 'project plans'

Comment (slandovs) [reply _episodes/01-plone.md]: I think there should also be a comma before 'if relevant'

File: _episodes/03-inventory.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ In 2017, OTN deployed the [SnipeIT Inventory](https://ops.oceantrack.org/snipeit
 
 ## General Tasks
 
-OTN Field technicians utilise SnipeIT on a regular basis, mainly doing the following activities:
+OTN Field technicians utilise SnipeIT on a regular basis, mainly doing the following activities.
```
Comment (slandovs) [inline _episodes/03-inventory.md]: I actually think this should still be a colon

File: _episodes/03-inventory.md [hunk 4]
```diff
@@ -45,14 +45,14 @@ Checking in equipment:
 
 ### Changing status of equipment
 
-Sometimes, you need to change the status of equipment in order to checkout the asset. The status will also need to be changed for units which are being shipped for RMA.
+Sometimes, you need to change the status of equipment in order to checkout the asset. The status will also need to be changed for units which are being shipped for RMA, or are lost.
```
Comment (slandovs) [inline _episodes/03-inventory.md]: no comma before "or are lost"

Comment (slandovs) [reply _episodes/03-inventory.md]: in my head, switching the order to "units which are lost or being shipped for RMA." sounds better?

File: _episodes/03-inventory.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ The mandatory columns and their formats are:
 - `checked out to`: project code - based on the list of `Users` in Snipe. can be blank if the asset is not yet assigned to a project.
 - `date shipped`: in format YYYY-MM-DD. can be blank
 - `expected return`: in format YYYY-MM-DD. can be blank
-- `notes`: freeform text. Please include transmitter ID (ex: A69-9001-12345) here if relevant
+- `notes`: freeform text. Please include transmitter ID (ex: A69-9001-12345) here if relevant (VR2ARs, VR2TXs, HR2s etc)
```
Comment (slandovs) [inline _episodes/03-inventory.md]: I struggle with the grammar around "if relevant"... should there be a comma before it? (at the very least - we are missing consistency around this)

File: _episodes/06-KPIdashboard.md [hunk 6]
```diff
@@ -13,9 +13,9 @@ OTNDC has developed an interactive web-based dashboard for Project Management st
 
 This curriculum is public, so we cannot add the login credentials here, but please email OTNDC@Dal.ca for information if you are interested.
 
-**Note**: this is for internal use only! The output from any of these KPI tools should not be shared externally without first discussing with Project Management or OTNDC.
+**Note**: this is for internal use only! The output from any of these KPI tools can **not** be shared externally without first discussing with Project Management or OTNDC.
```
Comment (slandovs) [inline _episodes/06-KPIdashboard.md:16]: **cannot**

File: _episodes/06-KPIdashboard.md [hunk 7]
```diff
@@ -46,18 +46,17 @@ The `Receiver Report Tool` has a direct link to the OTN Database, and can be use
 
 You are able to export as an XLSX, and soon as a CSV as well.
 
+**TIP: use this to trackdown shipping chaos - where has a receiver been reporting?**
```
Comment (slandovs) [inline _episodes/06-KPIdashboard.md]: track down (with a space)

File: _episodes/09-The Push.md [hunk 8]
```diff
@@ -20,13 +20,15 @@ Push events happen **three** times a year. They start on the third Thursday of t
 
 With the increased number of Nodes joining the Pushes, we are announcing the schedule for the next year. Please prepare in advance and mark your calendars.
 
-Push schedule through 2023:
+Example Push schedule through 2023:
```
Comment (slandovs) [inline _episodes/09-The Push.md]: should this be updated to 2024?

File: _episodes/11-obsolete-gitlab.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    - "What is the purpose of GitLab?"
+    - "How can the Field Team use GitLab?"
+keypoints:
+    - "GitLab helps organize and archive tasks"
+    - "There are several ways GitLab can be used by the Field Team"
+---
+
+## OTN's GitLab
+
+At OTN, GitLab is used for internal record keeping and collaborative code development. The main purpose (for non-computer scientists at OTN) is the ability to use GitLab Issues containing templates of task-lists to ensure the DAQ team NEVER forgets a step in data loading, and that no file is ever lost/forgotten in an inbox. We track all communications with researchers and archive all important information for future use.
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md:17]: should task-lists be hyphenated?

File: _episodes/11-obsolete-gitlab.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+GitLab allows us to be accountable, thorough, and comprehensive in our data-loading process. This is essential for smooth communication and acts as a searchable archive of OTN's institutional memory.
+
+## Field Ops Gitlab
+
+As the Field team has grown, job descriptions have expanded, and remote work has become more common it is now important to start using some of the lessons learned by the Data team to improve Field Ops communications. For this reason, there is now a **Field Team GitLab** project - [https://gitlab.oceantrack.org/otnfield/OTN_Field_Ops](https://gitlab.oceantrack.org/otnfield/OTN_Field_Ops).
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: ...more common, it is now

File: _episodes/11-obsolete-gitlab.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+Each OTN Field team member should have a GitLab account created - please fill out this [signup form for an account on GitLab](https://gitlab.oceantrack.org/users/sign_up) if you don't have an account already.
+
+This Field Ops Gitlab has been created as a place to track complex / important / outstanding issues so that they can be contributed to collaboratively and archived for future reference. This will be used in addition to Slack to make sure all Field Team members are communicating and information does not slip between the cracks of our busy schedules.
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: ...cracks with our busy...

File: _episodes/11-obsolete-gitlab.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+
+Gitlab Issues should be created to track:
+
+- lost equipment
+- equipment returns
+- found equipment
+- RMAs, including the outcome as reported by the manufacturer, and all data files (these usually are just emailed)
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: (these are usually just emailed)

File: _episodes/11-obsolete-gitlab.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+
+Current members of this Gitlab Project include: all Field staff, members of the Project Management Office, some members of the Data Team. Please tag them in an issue (using the @ symbol) to prompt them for input.
+
+Field team members leave (for vacation, or permanently) and so a shared record of important events will be essential to ensure knowledge-transfer. Having the information in your email is a great first step, but it may become lost or buried and is not easily shared/searched by colleagues.
+
+![FieldOps GitLab Home Page](../fig/fieldops_gitlab_home.PNG)
+
+## How to Use GitLab
+
+Menu on the left-hand sidebar:
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md:47]: mix of ending with punctuation vs no punctuation (not sure if we care about this)

File: _episodes/11-obsolete-gitlab.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+- Shows your main `Menu`, with all the GitLab projects of which you are a member
+- Has a `search` function, which can be used to find Issues using keywords
+- Has your personal `Issue Dashboard` with all Issues assigned to you
+- Has personal and `account` Settings
+
+**You will use the `Issues` and `Wiki` left-hand sidebar options, and the personal `Issue Dashboard` in the top menu banner (right side) most often.**
+
+## Issue Templates
+
+For tasks that are repeated (Canadian shipments, international shipments, returning equipment, washups, Fedex insurance claims etc) the same set of steps are often required each time. GitLab Issues have an option to populate with a standardized `template checklist`.
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: ...insurance claims, etc)

File: _episodes/11-obsolete-gitlab.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+
+Currently, the Field Ops GitLab only has one template, for `found_gear`, and some common `shipping` tasks. However, if this is deemed successful we can expand.
+
+### Using Issue Templates
+
+When you navigate to `Issues` on the left-hand sidebar menu, there will be an option to create a `New Issue`. By choosing the New Issue button in the top-right of your screen, you will be taken to a new, blank, issue form. To fill out the fields you will need to do the following:
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: "...new, blank issue form" remove the comma after blank

File: _episodes/11-obsolete-gitlab.md [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+- Title (something informative, ex: HFX washup 2022-04)
+- Type: No need to edit this field, should be type `Issue`.
+- Description: If if you wish to use a pre-made checklist `Template`, you will choose it here, using the drop down menu. Ensure you choose the relevant checklist for the type of issue you are creating (ex: found_gear). This will populate the large description field! 
+- Milestone: These can be setup to be deadlines, but can be left blank
+- Labels: This is for your reference - choose a label that will help you remember the reason this issue exists. Some common examples include `FOUND GEAR`, `Data/Metadata Problem`, `Legal`, `RMA` etc. You can create new labels at any time to help categorize the Issues!
```
Comment (slandovs) [inline _episodes/11-obsolete-gitlab.md]: comma after 'RMA' and before etc

File: _episodes/11-obsolete-gitlab.md [hunk 17]
```diff
@@ -0,0 +1,94 @@
+---
```
Comment (CaitlinBate) [inline _episodes/11-obsolete-gitlab.md:1]: NOTE: This lesson is obsolete (hence the re-naming). keeping it for posterity, but it is no longer relevant. No other content has been changed

File: _episodes/03-inventory.md (hunk 3 above)
Comment (naomitress) [reply _episodes/03-inventory.md]: i agree

File: _episodes/03-inventory.md (hunk 5 above)
Comment (naomitress) [reply _episodes/03-inventory.md]: how about "Please include transmitter ID (ex: A69-9001-12345) here, when relevant"

File: _episodes/06-KPIdashboard.md (hunk 6 above)
Comment (naomitress) [reply _episodes/06-KPIdashboard.md:16]: i think this is ok as the emphatic tone requires NOT to be in bold

File: _episodes/09-The Push.md (hunk 8 above)
Comment (naomitress) [reply _episodes/09-The Push.md]: @CaitlinBate please add the 2024 and future dates

**Review by naomitress — APPROVED** [review]
Approved, with the changes in comments from Shannon and a couple from me

File: _episodes/05-asana.md [hunk 18]
```diff
@@ -1,94 +1,87 @@
...
-Field team members leave (for vacation, or permanently) and so a shared record of important events will be essential to ensure knowledge-transfer. Having the information in your email is a great first step, but it may become lost or buried and is not easily shared/searched by colleagues.
+This Field Ops Asana has been created as a place to track complex / important / outstanding issues so that they can be contributed to collaboratively and archived for future reference. This will be used in addition to Slack to make sure all Field Team members are communicating and information does not slip between the cracks of our busy schedules.
 
-![FieldOps GitLab Home Page](../fig/fieldops_gitlab_home.PNG)
+Asana projects include:
 
-## How to Use GitLab
+- **Mission Planning:** forecast for upcoming missions, the field lead, and the required details needed leading up to the mission.
+- **Field Shipping:** track the incoming and outgoing OTN owned equipment shipments that are overseen by the field team. 
+- **Procurement:** keep track of equipment and consumables that need ordering, are incoming or have been received
```
Comment (naomitress) [inline _episodes/05-asana.md]: some items in this list have punctuation, others do not

Comment (naomitress) [inline _episodes/05-asana.md]: some items in this list have punctuation, others do not

**Pull request merged**